get_number.rb

```diff
@@ -0,0 +1,37 @@
+puts "Welcome to 'Get My Number!"
+
+print "What's your name? "
+input = gets
+name = input.chomp
+puts "Welcome, #{name}!"
+
+puts 'I have got a random number between 1 and 100.'
+puts 'Can you guess it?'
+target = rand(1..100)
+
+num_guesses = 0
+
+guessed_it = false
+
+until num_guesses == 10 || guessed_it
+
+  puts "You have got #{10 - num_guesses} guesses left."
+  print 'Make a guess: '
```

Comment:
https://docs.rubocop.org/rubocop/usage/auto_correct.html

Reply:
I used it